website/src/css/customTheme.scss

```diff
@@ -591,7 +591,7 @@ a[class*="tagRegular"] {
     &.dropdown {
       a {
         font-weight: 400;
-        font-size: 14px;
+        font-size: 17px;
```

Comment:
This changes the version dropdown `font-size` too, it's better to adjust the rule to match only version dropdown, or add an overwrite for the menu dropdowns only.

Also 17px seems a little too big for the dropdown entries, I suggest to go with 1rem, since the default size (.875rem) seems definitely bit small.